### PR TITLE
Doc:Text tweak Ver4.4.1

### PR DESCRIPTION
**Pull request by wallsman**
Text fix(一部テキスト修正) / Text tweak(リードミー回りの文章整形) /  4.4.1 Add Depot Recognition Line(倉庫アイテム認識の説明追加) / Screenshot change for 4.4.1

### CODE REVIEW

**Review by Tastror — APPROVED** [review]

File: README_ja-JP.md [hunk 1]
```diff
@@ -75,23 +75,27 @@ MAAは、MAA Assistant Arknightsです。
...
+  作戦、自動公開求人、戦友訪問、レワード収集、統合戦略（ベータ版）、公開求人認識の機能がサポートされている。
+  <br>サポート内容については[README](resource/global/YoStarJP/readme.md)を参照してください。
 - 韓国サーバー 
-  基本的な作戦機能がサポートされている。[README](resource/global/YoStarKR/readme.md)を参照してください。
+  基本的な作戦機能のみサポートされている。
+  <br>サポート内容については[README](resource/global/YoStarKR/readme.md)を参照してください。
 - TXWYサーバー  
-  基本的な作戦、自動公開求人、公開求人認識の機能がサポートされている。[README](resource/global/txwy/readme.md)を参照してください。
+  基本的な作戦、自動公開求人、公開求人認識の機能がサポートされている。
+  <br>サポート内容については[README](resource/global/txwy/readme.md)を参照してください。
```
Comment (MistEO) [inline README_ja-JP.md:98]: In markdown format, you can add two spaces at the end of a line instead of a `<br>` to indicate a line break.

**Comment by MistEO** [comment]
@Tastror 我刚在编辑评论呢，那你来改（

**Comment by Tastror** [comment]
@wallsman Could you give another sample screenshot, which have more logs on the right side?（たとえば、複数の作戦の後のようなログ）

**Comment by wallsman** [comment]
OK, change after.

**Pull request merged**